src/meltano/core/tracking/schemas/environment_context.json

```diff
@@ -0,0 +1,184 @@
+{
+    "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+    "description": "Schema for a Snowplow context describing the environment in which Meltano is operating",
+    "self": {
+        "vendor": "com.meltano",
+        "name": "environment_context",
+        "format": "jsonschema",
+        "version": "1-0-0"
+    },
+    "type": "object",
+    "properties": {
+        "context_uuid": {
+            "description": "A UUID that uniquely identifies an instance of this context",
+            "type": "string",
+            "pattern": "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$",
+            "minLength": 36,
+            "maxLength": 36
+        },
+        "meltano_version": {
+            "description": "The version of Meltano in use",
+            "type": "string",
+            "maxLength": 256
+        },
+        "is_dev_build": {
+            "description": "Whether the installed version of Meltano is a dev build",
+            "type": "boolean"
+        },
```

Comment:
Can we tell `is_dev_build` with certainty?

Reply:
Since I see the marker file is there. I think 'yes'.

Reply:
What we learn is whether the installed version came from `pypi.org` (or e.g. conda-forge), or if it came from someone building/installing from source. Upon reflection, as the PR is currently, it'll also be true for the version of Meltano found in our official Docker images.

Reply:
Thanks for the context, @WillDaSilva - For this, and for anything else that's kind of 'nice to have' but causing hangups, I think we can safely trim those from scope if needed and bring back in a future iteration.